frontend/components/Home/Dashboard/Forms/CreateCourseForm.tsx

```diff
@@ -53,9 +68,21 @@ const CreateCourseForm = (props: CreateCourseFormProps) => {
                 <Form.Input
                     id="course-title"
                     name="courseTitle"
-                    onChange={changeFunc}
+                    value={input.courseTitle}
+                    onChange={handleCourseTitleChange}
                     placeholder="Data Structures and Algorithms"
                 />
+                <div
+                    style={{
+                        textAlign: "right",
+                        color:
```

Comment:
This is default color (black) when you don't hit the char limit right?

Not sure if you already knew but, you can also do fancy syntax like
style={{
textAlign: "right",
...courseTitleCharCount < COURSE_TITLE_CHAR_LIMIT && { color: crimson }
}}